packages/models-library/src/models_library/folders.py

```diff
@@ -0,0 +1,5 @@
+from typing import TypeAlias
+
+from pydantic import PositiveInt
+
+FolderID: TypeAlias = PositiveInt
```

Comment:
I really wonder if we should not make use of NewType instead of TypeAlias all over the place:
see https://stackoverflow.com/questions/63318719/difference-between-type-alias-and-newtype